lib/concuss/headers.rb

```diff
@@ -30,7 +30,6 @@ class Concuss::Headers
     'Range',
     'Referer',
     'TE',
-    'User-Agent',
```

Comment:
I'm assuming that since we want to set a custom user agent, we can't test against user agent as a header when doing a scan right?

Reply:
Yes. Testing against User-Agent will require additional logic.

Reply:
Got it. I think that's fine for a later feature. Will get this built and released.